hollow/src/main/java/com/netflix/hollow/api/consumer/incubating/HollowClientUpdaterV2.java

```diff
@@ -0,0 +1,29 @@
+package com.netflix.hollow.api.consumer.incubating;
+
+import static java.util.stream.Collectors.toList;
+
+import com.netflix.hollow.api.client.HollowClientUpdater;
+import com.netflix.hollow.api.custom.HollowAPI;
+import java.util.stream.Collectors;
+
+public class HollowClientUpdaterV2<A extends HollowAPI> extends HollowClientUpdater {
```

Comment:
`FooV2` classes under an `incubating` package so that I could isolate these changes as much as possible while I noodled, not that I necessarily want to release them named as such.